<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>u3HClass</class>
 <widget class="QMainWindow" name="u3HClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>872</width>
    <height>374</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>u3H - Remote Access</string>
  </property>
  <property name="windowIcon">
   <iconset resource="u3h.qrc">
    <normaloff>:/u3H/u3hrsc/Acidrums4-Betelgeuse-Apps-rekonq.ico</normaloff>:/u3H/u3hrsc/Acidrums4-Betelgeuse-Apps-rekonq.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>5</number>
    </property>
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabPosition">
       <enum>QTabWidget::South</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="documentMode">
       <bool>false</bool>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <property name="movable">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="tab_clients">
       <attribute name="icon">
        <iconset resource="u3h.qrc">
         <normaloff>:/u3H/u3hrsc/users.png</normaloff>:/u3H/u3hrsc/users.png</iconset>
       </attribute>
       <attribute name="title">
        <string>Clients</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_tabClients">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QTreeWidget" name="treeWidget_clients">
          <property name="contextMenuPolicy">
           <enum>Qt::CustomContextMenu</enum>
          </property>
          <property name="indentation">
           <number>2</number>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <property name="animated">
           <bool>true</bool>
          </property>
          <attribute name="headerCascadingSectionResizes">
           <bool>false</bool>
          </attribute>
          <attribute name="headerDefaultSectionSize">
           <number>99</number>
          </attribute>
          <attribute name="headerHighlightSections">
           <bool>false</bool>
          </attribute>
          <attribute name="headerShowSortIndicator" stdset="0">
           <bool>true</bool>
          </attribute>
          <column>
           <property name="text">
            <string>ID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Wan</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Computer Name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Username</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Privileges</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Operating System</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_dashboard">
       <attribute name="icon">
        <iconset resource="u3h.qrc">
         <normaloff>:/u3H/u3hrsc/dashboard.png</normaloff>:/u3H/u3hrsc/dashboard.png</iconset>
       </attribute>
       <attribute name="title">
        <string>DashBoard</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox_listenersControl">
          <property name="title">
           <string>Listeners</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <widget class="QPushButton" name="pushButton_start_listeners">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Start Listeners</string>
             </property>
             <property name="icon">
              <iconset resource="u3h.qrc">
               <normaloff>:/u3H/u3hrsc/connect.png</normaloff>:/u3H/u3hrsc/connect.png</iconset>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="pushButton_stop_listeners">
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="text">
              <string>Stop Listeners</string>
             </property>
             <property name="icon">
              <iconset resource="u3h.qrc">
               <normaloff>:/u3H/u3hrsc/disconnect.png</normaloff>:/u3H/u3hrsc/disconnect.png</iconset>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="2">
            <widget class="QTreeWidget" name="treeWidget_listeners">
             <property name="indentation">
              <number>2</number>
             </property>
             <property name="headerHidden">
              <bool>true</bool>
             </property>
             <attribute name="headerVisible">
              <bool>false</bool>
             </attribute>
             <attribute name="headerDefaultSectionSize">
              <number>75</number>
             </attribute>
             <column>
              <property name="text">
               <string notr="true"/>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Port</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Status</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1" rowspan="2">
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Event Logs</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="0">
            <widget class="QTextEdit" name="textEdit_eventsLogs"/>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Stats</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Clients connected:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="label_dashboard_clientsCount">
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Uploaded bytes :</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="label_dashboard_uploaded">
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Downloaded bytes :</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLabel" name="label_dashboard_downloaded">
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_settings">
       <attribute name="icon">
        <iconset resource="u3h.qrc">
         <normaloff>:/u3H/u3hrsc/setting_tools.png</normaloff>:/u3H/u3hrsc/setting_tools.png</iconset>
       </attribute>
       <attribute name="title">
        <string>Settings</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item row="2" column="1">
         <widget class="QFrame" name="frame">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>25</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="leftMargin">
            <number>2</number>
           </property>
           <property name="topMargin">
            <number>2</number>
           </property>
           <property name="rightMargin">
            <number>2</number>
           </property>
           <property name="bottomMargin">
            <number>2</number>
           </property>
           <item>
            <widget class="QPushButton" name="pushButton_settingPage_resetDefault">
             <property name="text">
              <string>Reset to default</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>291</width>
               <height>16</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_settingPage_saveSettings">
             <property name="text">
              <string>Save Settings</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="0" rowspan="3">
         <widget class="QTreeWidget" name="treeWidget_settings">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="indentation">
           <number>2</number>
          </property>
          <attribute name="headerVisible">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string>0</string>
           </property>
          </column>
          <item>
           <property name="text">
            <string>General</string>
           </property>
           <property name="icon">
            <iconset resource="u3h.qrc">
             <normaloff>:/u3H/u3hrsc/settings.png</normaloff>:/u3H/u3hrsc/settings.png</iconset>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QStackedWidget" name="stackedWidget_settings">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="page_listeners">
           <widget class="QWidget" name="gridLayoutWidget">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>10</y>
              <width>171</width>
              <height>61</height>
             </rect>
            </property>
            <layout class="QGridLayout" name="gridLayout_8">
             <item row="0" column="0">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Connexions port : </string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="lineEdit_connexionPort"/>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Transferts port : </string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="lineEdit_transfertsPort"/>
             </item>
            </layout>
           </widget>
           <widget class="QCheckBox" name="checkBox_autostartlisteners">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>80</y>
              <width>151</width>
              <height>21</height>
             </rect>
            </property>
            <property name="text">
             <string>Start listeners on start</string>
            </property>
           </widget>
           <widget class="QCheckBox" name="checkBox_enableVerboseDashboard">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>250</y>
              <width>191</width>
              <height>21</height>
             </rect>
            </property>
            <property name="text">
             <string>Enable debug output in dashboard</string>
            </property>
           </widget>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_transferts">
       <attribute name="icon">
        <iconset resource="u3h.qrc">
         <normaloff>:/u3H/u3hrsc/download.png</normaloff>:/u3H/u3hrsc/download.png</iconset>
       </attribute>
       <attribute name="title">
        <string>Transferts</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="0" column="0">
         <widget class="QTableWidget" name="tableWidget_transferts">
          <column>
           <property name="text">
            <string>Client</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Filename</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Size</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Progress</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>872</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <property name="icon">
     <iconset resource="u3h.qrc">
      <normaloff>:/u3H/u3hrsc/menu.png</normaloff>:/u3H/u3hrsc/menu.png</iconset>
    </property>
    <widget class="QMenu" name="menuTheme">
     <property name="title">
      <string>Theme</string>
     </property>
    </widget>
    <addaction name="menuTheme"/>
    <addaction name="separator"/>
    <addaction name="actionCreate_Server"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionClose_listener">
   <property name="text">
    <string>Close listener</string>
   </property>
  </action>
  <action name="actionCreate_Server">
   <property name="text">
    <string>Create Server</string>
   </property>
  </action>
  <action name="actionA">
   <property name="text">
    <string>a</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="u3h.qrc"/>
 </resources>
 <connections/>
</ui>
